TT_REST_API/FillDownload/python/fill_download.py

```diff
@@ -282,7 +282,7 @@ def retrieve_fills(environment, headers, min_time_stamp=None, max_time_stamp=Non
     if max_time_stamp:
         fill_download_url += '?maxTimestamp={}'.format(max_time_stamp)
     fills = api_request(fill_download_url, headers, request_timeout=True)
-    if fills.status_code == 408:
+    if type(fills) is not dict and fills.status_code == 408:
```

Comment:
I am not sure about this code. Why don't you make the function always return the decoded response or the raw response.